css-layout-classes/styles.css

```diff
@@ -0,0 +1,53 @@
+#container {
+  max-width: 1000px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.row {
+  display: flex;
+  width: 100%;
+  height: 100%;
+}
+
+.column-one-third {
+  flex-basis: calc(100%/3);
+}
+
+.column-two-thirds {
+  flex-basis: 66.6%;
+}
+
+.column-full {
+  flex-basis: calc(100%/1);
```

Comment:
In this case, it doesn't need any calculation so it can be just `100%` without `calc`